jcl/src/java.base/share/classes/com/ibm/oti/util/Util.java

```diff
@@ -1,12 +1,8 @@
 /*[INCLUDE-IF Sidecar16]*/
```

Comment:
Since we are here, please change it to `Sidecar18-SE`, other than that, it looks good.

Reply:
Any reason not to use `JAVA_SPEC_VERSION >= 8`?

Reply:
That works as well.